Source/WebKit/NetworkProcess/NetworkConnectionToWebProcess.cpp

```diff
@@ -911,6 +932,11 @@ void NetworkConnectionToWebProcess::allCookiesDeleted()
 
 #endif
 
+void NetworkConnectionToWebProcess::cookieEnabledStateMayHaveChanged()
+{
+    connection().send(Messages::NetworkProcessConnection::UpdateCachedCookiesEnabled(), 0);
```

Comment:
Please add a `protectedConnection()` getter which returns a smart pointer and call it here.

Reply:
I find there are many places using `connection()` in this file, so let me land this patch first and I will do a follow-up to change all of them: https://bugs.webkit.org/show_bug.cgi?id=268085.

Reply:
Please do the change I suggest in this patch. You don't have to fix ALL cases in this PR, just this one.

Reply:
Added.

Reply:
Thank you.